### PR TITLE
Added Expressive code section to Emergence

### LINKED ISSUE #139: Book Club: Document our use of "Clean Code: Emergence: Expressive"
Expressive

- Follow rules from “Meaningful Names” chapter
- Keep functions, classes and names as small as it makes sense to be
- Take the time to review to make sure that it makes sense for the next person
- Every part of the inputs and outputs matter
  - ex: use ISet instead of IEnumerable or List when possible

### PR DESCRIPTION
**Pull request by klauffer**
resolves #139

### CODE REVIEW

File: clean-code/EMERGENCE.md [hunk 1]
```diff
@@ -18,7 +18,29 @@ TODO https://github.com/AndcultureCode/AndcultureCode/issues/138
 
 ## Expressive
 
-TODO https://github.com/AndcultureCode/AndcultureCode/issues/139
+Expressive code comes in many forms. the most obvious ways for code to be expressive is to use variable names that inform the reader of exactly what the symantec meaning is behind its purpose. However, there are more ways to make code expressive then just that. For example languages that have a paradigm of strong types allows for expressive code to be built directly into the types. For Instance, Let's take the following interface
```
Comment (rllowman) [inline clean-code/EMERGENCE.md:21]: Comment
```suggestion
Expressive code comes in many forms. the most obvious ways for code to be expressive is to use variable names that inform the reader of exactly what the semantic meaning is behind its purpose. However, there are more ways to make code expressive then just that. For example languages that have a paradigm of strong types allows for expressive code to be built directly into the types. For Instance, Let's take the following interface
```

Comment (jstevenson) [reply clean-code/EMERGENCE.md:21]: Please add a period or colon to the end of the last sentence.

File: clean-code/EMERGENCE.md [hunk 2]
```diff
@@ -18,7 +18,29 @@ TODO https://github.com/AndcultureCode/AndcultureCode/issues/138
...
+
+```C#
+public interface IDoSomething
+{
+    IEnumerable<string> GetEmails(string Id);
+}
+```
+
+what can we gather from this interface definition? It says its getting emails and it requires a certain Id. However that isnt enough information for a developer to be able to determine what its actually doing. what Id does it need? is the list unique? are the strings returned valid emails? These questions arise quickly as you are attempting to use the method. How can we make it better?
```
Comment (jstevenson) [inline clean-code/EMERGENCE.md:30]: Capitalize the beginning of sentences and add the apostrophe in "isnt".

File: clean-code/EMERGENCE.md [hunk 3]
```diff
@@ -18,7 +18,29 @@ TODO https://github.com/AndcultureCode/AndcultureCode/issues/138
...
+Lets take the following refactored example
+
+```C#
+public interface IDoSomething
+{
+    ISet<Email> GetEmails(EmailGroupId Id);
+}
+```
+
+This refactored code leans into strong types to inform the developer of exactly the intent. Now we can say that its getting a unique set of valid emails from the `ISet<Emails>`. we can also say that the Id is a EmailGroupId and not just any string.
```
Comment (jstevenson) [inline clean-code/EMERGENCE.md:41]: Capitalize "we"

**Review by jstevenson — APPROVED** [review]
Looks good other than some minor typos.

**Pull request merged**